.github/workflows/tox-pytest.yml

```diff
@@ -1,14 +1,99 @@
 ---
 name: tox-pytest
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Good question, I ended up puzzling over the docs for a bit before making this decision!

We definitely want `pull_request` since it [runs checks on the merged version](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request).

The benefit of keeping `push` is that we get CI without having to make a PR. Drawback is getting a lot of noise when you're looking at PR checks. If we were paying for CI, that would be another drawback 😉 (thank you, MSFT, for your benevolence).

IMO the tradeoff of "have to make a PR if you want CI to run" for "fewer check marks" is worth it - especially if we're trying to incentivize early draft PR creation anyways.

Reply:
Seems good.